### DISCUSSION TITLE
Clocks

### DISCUSSION THREAD

**Discussion by vyPal** [DigitalLogicSimCommunity/Digital-Logic-Sim-CE#6] [Ideas]
You could add clocks for no obvious reason. If you want to I modified your latest code to add a 1Hz and 2Hz clock into the simulator. I can upload the modified version to github and make a pul request if you think clocs  would be a good adition.

**Comment by AOx0** [comment]
Hm, maybe a `x` Hz configurable clock would be a nice alternative to the traditional https://github.com/SebLague/Digital-Logic-Sim/pull/66#issuecomment-858147493. I don't have that much time. So, if you are willing to develop it, your PR is welcomed.

**Reply by vyPal** [reply]
Ok I'll try making a counfigurable clock
